docker/ci/docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: "3.4"
+services:
+  armadillo:
+    hostname: armadillo
+    ports:
+      - 8080:8080
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+    image: molgenis/molgenis-armadillo:latest
+    environment:
+      LOGGING_CONFIG: 'classpath:logback-file.xml'
+      AUDIT_LOG_PATH: '/app/logs/audit.log'
+      SPRING_SECURITY_USER_PASSWORD: 'admin'
+    volumes:
+      - ${PWD}/fake-tree/armadillo/logs:/logs
+      - ${PWD}/fake-tree/armadillo/data:/data
+      - ${PWD}/fake-tree/armadillo/config:/config
```

Comment:
Just tested local ... 

```sh
cd docker/ci
../bin/prepare.bash ci
docker compose up

docker container run --network container:ci-armadillo-1 \
              --interactive --tty \
              --entrypoint /bin/bash molgenis/r-cicd -c "ls -l / ; cd /cicd/scripts/release ; ./armadillo-ready.bash"
```

succeeds (1/3)

We think the volumes  do mount on local dev but not on CircleCI somehow.

Reply:
See https://circleci.com/docs/docker-compose/#using-docker-compose-with-docker-executor